r-package/grf/tests/testthat/test_clustered_standard_errors.R

```diff
@@ -0,0 +1,50 @@
+library(grf)
+
+set.seed(4321)
+
+test_that("Clustered standard errors are greater than unclustered", {
+
+    cluster_size <- 20
+    # data sim
+    X <- rnorm(1000)
```

Comment:
Please define the simulation in terms of `n`, `p`, etc. Also, forests may behave very strangely when `p = 1`, so I'd recommend using at least `p = 4`.

Also, to make sure running tests doesn't slow down development, it might be better to use smaller sample sizes if they still highlight the desired phenomena (e.g., I'm using `n=200; p = 4; cluster_size = 10` and the results are qualitatively similar):

```
> c(mean_no_cluster, mean_uncorrected, mean_corrected, mean_corrected_no_cluster)
[1] 0.06357332 0.02013494 0.15627015 0.12970989
```